src/monte_carlo.c

```diff
@@ -0,0 +1,249 @@
+#include "monte_carlo.h"
+#include "linear_congruential_random_generator.h"
+#include <math.h>
+#include <stdlib.h>
+#include <string.h>
+#include <time.h>
+
+#ifndef DRAW
```

Comment:
Do we want a user to be able to define these values themselves? If we do not I think it might be a good idea to remove the ifndef.